docs/core/event_handler/bedrock_agents.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Agents for Amazon Bedrock
+description: Core utility
+---
+
+Author [Agents for Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html#agents-how){target="_blank"} using event handlers and auto generation of OpenAPI schemas.
+
+<center>
+```mermaid
+--8<-- "docs/core/event_handler/bedrock_agents.mermaid"
+```
+</center>
+
+## Key features
+
+* Similar experience when authoring [REST](api_gateway.md){target="_blank"} and [GraphQL APIs](appsync.md){target="_blank"}
+* Minimal boilerplate to build Agents for Amazon Bedrock
+* Automatic generation of [OpenAPI schemas](https://www.openapis.org/){target="_blank"} from your business logic code
+* Built-in data validation for requests and responses
+
+## Terminology
+
+**Data validation** automatically validates the user input and the response of your AWS Lambda function against a set of constraints defined by you.
+
+**Event handler** is a Powertools for AWS feature that processes an event, runs data parsing and validation, routes the request to a specific function, and returns a response to the caller in the proper format.
```

Comment:
I think this should be a bullet list, it'll make it easier to parse for ESL folk

Reply:
This follows the same style used in other recent documentation, for instance [Data Masking](https://docs.powertools.aws.dev/lambda/python/latest/utilities/data_masking/)